Dockerfile.hermetic

```
@@ -2,13 +2,16 @@
 # Stage 1: Build stage
 FROM registry.access.redhat.com/ubi9/nodejs-22:9.5-1746535891 AS builder
 
+# Additional arguments to be appended for `npm ci`, e.g. "--legacy-peer-deps"
+ARG NPM_CI_ARGS=
```

Let's use the provided value if present, if not let's set it to empty string
```suggestion
export NPM_CI_ARGS=${NPM_CI_ARGS-""}
```